supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -348,6 +348,28 @@ There is no functional difference between the first server that was installed an
 
 *See also*: xref:bucket-index[bucket index]
 
+[[inference]]
+==== image:images/yes.png[yes] inference (noun)
+*Description*: The act a model generating outputs from input data. For example, "Inference speeds increased on the new models"
+
+*Use it*: yes
+
+[.vale-ignore]
+*Incorrect forms*: 
+
+*See also*:
+
+[[inferencing]]
```

Comment:
```suggestion
[[inference serving]]
```

Reply:
"Inference serving" seems to be the proper form.

Reply:
They seem to both be correct terms, but they have different meanings.